tools/phpunit/phpunit.xml

```diff
@@ -23,6 +23,7 @@
             <exclude>
                 <directory>./../../vendor/</directory>
                 <directory>./../../tests/</directory>
+                <file>./../../lib/SimpleSAML/Utils/HttpAdapter.php</file>
```

Comment:
For some reason, the code coverage process was trying to include this file multiple times, which was killing it.  So the simplest thing to do was to ignore it (it doesn't need to be included in any case...).